docs/src/Box.doc.js

```diff
@@ -1,24 +1,154 @@
+    <MainSection.Subsection
+      description={`The following examples are all built using the \`Box\` component and its various properties. `}
+    >
+      <MainSection.Card
+        title="Media Object"
+        description={`
+           The [media object](http://www.stubbornella.org/content/2010/06/25/the-media-object-saves-hundreds-of-lines-of-code/) is a common pattern for displaying data. What's interesting about this example is the use of \`flex\` to align the items. If you try changing the size of the \`Avatar\` or the number of lines of \`Text\`, both will stay aligned because they are center aligned.
+
+           In the double-sided example we use \`flex="grow"\` to mark a flex child as something that can expand. Try removing the \`grow\` property and see what happens.
+
+        `}
+        cardSize="lg"
+        defaultCode={`
+<Box column={10}>
+  <Box
+    alignItems="center"
+    direction="row"
+    display="flex"
+    width={300}
+    marginStart={-1}
+    marginEnd={-1}
+    paddingY={2}
+  >
+    <Box paddingX={1}>
+      <Avatar name="chrislloyd" size="md" />
+    </Box>
+    <Box paddingX={1}>
+      <Text weight="bold">Chris Lloyd</Text>
+      <Text>joined 2 years ago</Text>
+    </Box>
+  </Box>
+  <Box
+    alignItems="center"
+    direction="row"
+    display="flex"
+    marginStart={-1}
+    marginEnd={-1}
+  >
+    <Box paddingX={1}>
+      <Avatar name="chrislloyd" size="md" />
+    </Box>
+    <Box paddingX={1} flex="grow">
+      <Text weight="bold">Chris Lloyd</Text>
+      <Text>joined 2 years ago</Text>
+    </Box>
+    <Box paddingX={1}>
+      <Button text="Follow" size="sm" color="red" />
+    </Box>
+  </Box>
+</Box>
+`}
+      />
+      <MainSection.Card
+        cardSize="md"
+        title="Expandable Module"
+        description="Underneath the hood, [Expandable Module](/Module) uses Box to create the styling of the rounded edges and shadow, while also ensuring the title can grow while keeping the icon aligned to the end."
+        defaultCode={`
+  <Module.Expandable
+    id="ModuleExample3"
+    accessibilityExpandLabel="Expand the module"
+    accessibilityCollapseLabel="Collapse the module"
+    items={[
+      {
+        title: 'Example with icon',
+        children: <Text size="md">Children1</Text>,
+        iconAccessibilityLabel: 'title icon',
+        icon: 'lock',
+      },
+    ]}
+  />
+`}
+      />
+      <MainSection.Card
+        cardSize="md"
+        title="Stacked Checkboxes"
+        description="In this example, Box is used to create spacing between [Checkbox](/Checkbox) items, as well as set the height of the image in each item."
+        defaultCode={`
+<Box margin={-2}>
+  <Box padding={2}>
+    <Checkbox
+      checked={false}
+      id="coral"
+      label="Coral"
+      subtext="Botanical art in coral and green"
+      image={
+        <Box height={100} width={80}>
+          <Image
+            alt="Botanical art in coral and green"
```

Comment:
can we remove the spaces in here that push the indentation? 
![image](https://user-images.githubusercontent.com/10593890/106484748-e922e580-647d-11eb-9f24-69e0410fae23.png)